api/client/proto/authservice.proto

```diff
@@ -680,6 +693,30 @@ message GetMFADevicesRequest {}
 // GetMFADeviceResponse is a response for GetMFADevices RPC.
 message GetMFADevicesResponse { repeated types.MFADevice Devices = 1; }
 
+// UserSingleUseCertsRequest is a request for a single-use user certificate.
+message UserSingleUseCertsRequest {
+    oneof Request {
+        UserCertsRequest Init = 1;
+        MFAAuthenticateResponse MFAResponse = 2;
+    }
+}
+
+// UserSingleUseCertsResponse is a response with a single-use user certificate.
+message UserSingleUseCertsResponse {
+    oneof Response {
+        MFAAuthenticateChallenge MFAChallenge = 1;
+        UserCert Cert = 2;
+    }
+}
+
+// UserCert is a single user certificate, either SSH or TLS.
```

Comment:
Did you mean "single-use user certificate" or "single" as in "one"? If the latter, I'd just say "is a user certificate". If the former, prob. makes sense to rename the type to `SingleUseUserCert` or something, otherwise too generic.

Reply:
Updated to `SingleUseUserCert`